File: app/src/main/java/com/worldventures/dreamtrips/wallet/analytics/settings/ProfileChangesSavedAction.java
package com.worldventures.dreamtrips.wallet.analytics.settings;

import com.worldventures.core.service.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.dreamtrips.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:settings:general:smartcard profile:changes saved",
                trackers = AdobeTracker.TRACKER_KEY)
public class ProfileChangesSavedAction extends WalletAnalyticsAction {
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/settings/ProfileChangesSavedAction.kt
package com.worldventures.wallet.analytics.settings

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:settings:general:smartcard profile:changes saved", trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class ProfileChangesSavedAction : WalletAnalyticsAction()
